src/backends/arch.rs

```diff
@@ -17,11 +17,12 @@ pub struct ArchQueryInfo {
     pub explicit: bool,
 }
 
-pub struct ArchModification {
+#[derive(Debug, Clone)]
+pub struct ArchModificationOptions {
```

Comment:
Needs to implement `From<toml::map::Map<....`
See below

Reply:
Fixed in the other thread